### PR TITLE
Fix golint issues in pkg/credentialprovider/plugin

### PR DESCRIPTION
**Pull request by geegeea**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR fixes golint issues in `pkg/credentialprovider/plugin`

**Which issue(s) this PR fixes**:
Part of #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```

```

### CODE REVIEW

**Comment by geegeea** [comment]
@sttts @thockin Can someone please review? It's a tiny change. Alternatively, if there's something wrong with the PR, let me know and I'll close it.

File: pkg/credentialprovider/plugin/plugin.go [hunk 1]
```diff
@@ -252,7 +252,7 @@ func (p *pluginProvider) Provide(image string) credentialprovider.DockerConfig {
 }
 
 // Enabled always returns true since registration of the plugin via kubelet implies it should be enabled.
-func (e *pluginProvider) Enabled() bool {
+func (p *pluginProvider) Enabled() bool {
```
Comment (thockin) [inline pkg/credentialprovider/plugin/plugin.go:255]: FYI - if you don't use the arg at all, you can just leave out entirely :)

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**